server/bin/docker/standalone_test.json

```diff
@@ -173,7 +173,7 @@
               "downsampleAggregator": "max",
               "downsampleFillPolicy": "none",
               "downsampleInterval": "1s",
-              "metric": "sys.cpu0.user",
+              "metric": "sys.cpu.user",
```

Comment:
This has all been fixed and added as a standard dashboard to the timely app. see grafana/timely-app/src/datasource/dashboards/standalone_test.json